packages/test/test-service-load/testConfig.json

```diff
@@ -25,7 +25,7 @@
 				}
 			},
 			"optionOverrides": {
-				"odsp": {
+				"odsp-odsp": {
```

Comment:
The bug is that the test expects `odsp-odsp` instead of just one `odsp`. The test should be fixed because the extra odsp makes no sense (unless that fix is too complex).

Reply:
So it's based on "driver-endpoint". The only variations we do have is based on "endpoint" however.